OrbitQt/CutoutWidget.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_QT_CUTOUT_WIDGET_
+#define ORBIT_QT_CUTOUT_WIDGET_
+
+#include <QLabel>
+
+class CutoutWidget : public QLabel {
```

Comment:
A short comment stating the use-case of this widget and that it is related to the TutorialOverlay would be great.

Reply:
Good point, done.